lib/spack/spack/cmd/buildcache.py

```diff
@@ -234,7 +234,7 @@ def setup_parser(subparser):
     update_index.set_defaults(func=buildcache_update_index)
 
 
-def find_matching_specs(pkgs, allow_multiple_matches=False, env=None):
+def find_matching_specs(specs, allow_multiple_matches=False, env=None):
```

Comment:
This seems to be a good opportunity to add `env` to the docstring.  You could take the description from `find_matching_specs` in `cmd/uninstall.py`.

Reply:
Done